scania-digital-design-system/sdds
# feat(textarea/textfield): migrate textarea textfield component

**Describe pull-request**  
Migration of textarea and textfield component

**Solving issue**  
Fixes: [AB#2409](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2409) [AB#2410](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2410)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in components -> textarea/textfield
3. Check that the design is aligned with the one in Figma. 

**Screenshots**  
-

**Additional context**  
-
